charts/cryostat/templates/deployment.yaml

```diff
@@ -190,6 +190,14 @@ spec:
           securityContext:
             {{- toYaml .Values.storage.securityContext | nindent 12 }}
         {{- if not .Values.minimal }}
+        - name: {{ printf "%s-%s" .Chart.Name "auth-proxy" }}
+          securityContext:
+            {{- toYaml .Values.authProxy.securityContext | nindent 12 }}
+          image: "{{ .Values.authProxy.image.repository }}:{{ .Values.authProxy.image.tag }}"
+          imagePullPolicy: {{ .Values.authProxy.image.pullPolicy }}
+          volumeMounts:
+            - mountPath: /etc/oauth2_proxy
+              name: {{ auth-proxy-config }}
```

Comment:
I don't think this needs to be wrapped in curly braces - it's just a reference to the literal `name: auth-proxy-config` lower down in this same file. The curly braces are for variable interpolation.

```
$ helm install cryostat --set pvc.enabled=true --set core.route.enabled=true ./charts/cryostat
Error: INSTALLATION FAILED: parse error at (cryostat/templates/deployment.yaml:200): bad character U+002D '-'
```